src/core/Table/Table.tsx

```diff
@@ -262,6 +267,13 @@ export const Table = <
     onSelect?.(selectedData, newState);
   };
 
+  const onRowClickHandler = React.useCallback(
```

Comment:
It doesn't give you anything. If original `onRowClick` is not memoized then this one will also return new function reference every time.

Reply:
Maybe add comment above onRowClick with a recommendation to memo it?

Reply:
I was thinking of adding selection stuff here, instead of story. Because it is part of standards.

Reply:
Ahhh I can't get that to work without maintaining my own state, so I've removed it.

Reply:
Added back with some basic row selection